bot/check-result.sh

```diff
@@ -0,0 +1,498 @@
+else
+    # FAILURE
+    status="FAILURE"
+    summary=":cry: FAILURE"
+fi
+
+### Example details/descriptions
+# Note, final string must not contain any line breaks. Below examples include
+# line breaks for the sake of readability.
+# <details>
+#   <summary>:cry: FAILURE _(click triangle for detailed information)_</summary>
+#   Details:<br/>
+#   &nbsp;&nbsp;&nbsp;&nbsp;:heavy_check_mark: job output file <code>slurm-470503.out</code><br/>
+#   &nbsp;&nbsp;&nbsp;&nbsp;:heavy_multiplication_x: found message matching <code>ERROR: </code><br/>
+#   &nbsp;&nbsp;&nbsp;&nbsp;:heavy_multiplication_x: found message matching <code>FAILED: </code><br/>
+#   &nbsp;&nbsp;&nbsp;&nbsp;:heavy_multiplication_x: found message matching <code> required modules missing:</code><br/>
+#   &nbsp;&nbsp;&nbsp;&nbsp;:heavy_check_mark: found message(s) matching <code>No missing installations</code><br/>
+#   &nbsp;&nbsp;&nbsp;&nbsp;:heavy_check_mark: found message matching <code>tar.gz created!</code><br/>
+#   Artefacts:
+#   <li><code>eessi-2023.04-software-linux-x86_64-amd-zen2-1682384569.tar.gz</code></li>
+# </details>
+#
+# <details>
+#   <summary>:grin: SUCCESS _(click triangle for detailed information)_</summary>
+#   Details:<br/>
+#   &nbsp;&nbsp;&nbsp;&nbsp;:heavy_check_mark: job output file <code>slurm-470503.out</code><br/>
+#   &nbsp;&nbsp;&nbsp;&nbsp;:heavy_check_mark: found message matching <code>ERROR: </code><br/>
+#   &nbsp;&nbsp;&nbsp;&nbsp;:heavy_check_mark: found message matching <code>FAILED: </code><br/>
+#   &nbsp;&nbsp;&nbsp;&nbsp;:heavy_check_mark: found message matching <code> required modules missing:</code><br/>
+#   &nbsp;&nbsp;&nbsp;&nbsp;:heavy_check_mark: found message(s) matching <code>No missing installations</code><br/>
+#   &nbsp;&nbsp;&nbsp;&nbsp;:heavy_check_mark: found message matching <code>tar.gz created!</code><br/>
+#   Artefacts:
+#   <li><code>eessi-2023.04-software-linux-x86_64-amd-zen2-1682384569.tar.gz</code></li>
+# </details>
+#
+# <details>
+#   <summary>:grin: SUCCESS _(click triangle for detailed information)_</summary>
+#   <dl>
+#     <dt>_Details_</dt>
+#     <dd>
+#       :heavy_check_mark: job output file <code>slurm-4682.out</code><br/>
+#       :heavy_check_mark: no message matching <code>ERROR: </code><br/>
+#       :heavy_check_mark: no message matching <code>FAILED: </code><br/>
+#       :heavy_check_mark: no message matching <code> required modules missing:</code><br/>
+#       :heavy_check_mark: found message(s) matching <code>No missing installations</code><br/>
+#       :heavy_check_mark: found message matching <code>tar.gz created!</code><br/>
+#     </dd>
+#     <dt>_Artefacts_</dt>
+#     <dd>
+#       <details>
+#         <summary><code>eessi-2023.04-software-linux-x86_64-generic-1682696567.tar.gz</code></summary>
+#         size: 234 MiB (245366784 bytes)<br/>
+#         entries: 1234<br/>
+#         modules under _2023.04/software/linux/x86_64/intel/cascadelake/modules/all/_<br/>
```

Comment:
`intel/cascadelake` doesn't match tarball name which mentions `generic`, so let's change that, and bump to `2023.06` while we're at it?

Reply:
Updated.